### DISCUSSION TITLE
What is the process for submitting topics to Milvus blog?

### DISCUSSION THREAD

**Discussion by bjzhjing** [milvus-io/milvus#29976] [Q&A and General discussion]
Is there any link or document to tell how to submit articles to [Milvus blog](https://milvus.io/blog)? I cann't find an entry in that page.

**Comment by yhmo** [comment]
It is not open to the public, only official articles are posted. If you want to post your article, I can contact the maintainer.

**Comment by xiaofan-luan** [comment]
> Is there any link or document to tell how to submit articles to [Milvus blog](https://milvus.io/blog)? I cann't find an entry in that page.

please directly contact xiaofan.luan@zilliz.com and I'm happy to help

**Comment by bjzhjing** [comment]
Thanks @yhmo and @xiaofan-luan for your quick response! I will write a note to xiaofan for details discussion.

**Comment by bjzhjing** [comment]
Hello @xiaofan-luan Just want to confirm if you have received my email?
